src/random_test.cpp

```diff
@@ -1362,14 +1364,30 @@ std::string Table::definition() {
 
   def += " )";
   static auto no_encryption = opt_bool(NO_ENCRYPTION);
+  bool keyring_key_encrypt_flag = 0;
 
-  if (!no_encryption && type != TEMPORARY)
-    def += " ENCRYPTION='" + encryption + "'";
+  if (!no_encryption && type != TEMPORARY) {
```

Comment:
how this will work with oracle encryption style. IT doesn't accept ENCRYPTION_KEY_ID

Reply:
Fixed